### PR TITLE
add docs how to setup and how to run a simple test

### LINKED ISSUE #37: Add example yamls on how to test it.
We should provide example yamls for an application for the end user to test this in the suggested scenario.

### PR DESCRIPTION
**Pull request by szuecs**
fix #37 and make the deployment for #18 more clear

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Deploy
+
+## Requirements
+
+* You have a running Kubernetes Cluster on AWS.
+* You have a route53 Hosted Zone in your AWS account.
+* You have provisioned valid ACM or IAM certificates that are valid
```
Comment (mikkeloscar) [inline deploy/README.md]: One too many `valid` :)

File: deploy/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Install
+
+    % cd deploy
+    # install the ingress implementation
+    % kubectl create -f skipper.yaml
+    # install the controller which glues together AWS and the ingress implementation
+    % kubectl create -f ingress-controller.yaml
+
+If you have done this, you cant use our
```
Comment (mikkeloscar) [inline deploy/README.md]: cant -> can

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**